migration/src/main/java/kin/sdk/Logger.java

```diff
@@ -0,0 +1,41 @@
+package kin.sdk;
+
+import android.text.TextUtils;
+import android.util.Log;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public class Logger {
+
+    private static final String TAG = "KinMigrationModule";
+    private static boolean shouldLog;
+
+    public static void enable(boolean enable) {
+        shouldLog = enable;
+    }
+
+    public static void d(String message) {
```

Comment:
Add also w for Log.w

Reply:
let's add by demand, added what's needed now